wiki/Configuration:-Window-Rules.md

```diff
@@ -235,6 +256,21 @@ window-rule {
 }
 ```
 
+#### `default-window-height`
+
+<sup>Since: next release</sup>
+
+Set the default height for the new window if it opens as floating.
+
+```kdl
+// Make Alacritty take 50% of the screen height on opening.
+window-rule {
```

Comment:
Description above says it sets the height if it is opened as floating, but this rule doesn't apply to only floating windows.

Reply:
I actually intend to make it apply to all windows; currently it should only like half apply to scrolling ones I think